pubsublite/spark-connector/README.md

```diff
@@ -0,0 +1,184 @@
+# Using Spark SQL Streaming with Pub/Sub Lite
+
+The samples in this directory show how to read messages from and write messages to Pub/Sub Lite from an [Apache Spark] cluster using the [Pub/Sub Lite Spark Connector].
+
+Visit this Maven [link](https://search.maven.org/search?q=g:com.google.cloud%20a:pubsublite-spark-sql-streaming) to download the connector's uber jar. The uber jar has a "with-dependencies" suffix. You will need to include it on the driver and executor classpaths when submitting a Spark job, typically in the `--jars` flag. Alternatively, you can get this JAR from this [public Cloud Storage location](gs://spark-lib/pubsublite/pubsublite-spark-sql-streaming-LATEST-with-dependencies.jar).
```

Comment:
I would suggest using the GCS link of the latest jar so customer always get the latest one as the top suggestion instead of maven. Maven should be a way to view all versions and release notes.

Reply:
Reversed.